src/main/java/Duke/DukeSortInputTask.java

```diff
@@ -0,0 +1,33 @@
+package Duke;
+
+import java.util.ArrayList;
+
+public class DukeSortInputTask {
```

Comment:
Could there be any better naming than this? It is unclear what you are referring to with the word "SortInputTask".